tasks/37_min-number-to-prime/min-number-to-prime.js

```diff
@@ -0,0 +1,22 @@
+/**
+ * minNumberToPrime - find minimum number to be inserted
+ * in a list, so that sum of all elements of list should
+ * equal the closest prime number
+ *
+ * @param {Array<Number>} arr
+ * @return {Number}
+ */
+export function minNumberToPrime(arr) {
+  const sum = arr.reduce((acc, val) => acc + val, 0);
+  const isPrime = (num) => {
```

Comment:
The same as above. Do we really need to declare isPrime function here?

Reply:
Declaring isPrime function makes code more readable. That's why i decided declare isPrime.

Reply:
What @KalachevDev is talking about is that there's no need to declare it inside `minNumberToPrime` function. You can do it in outer scope to avoid creating new `isPrime` function on every call of `minNumberToPrime`.

Reply:
Thx for explanation. Now i understand what @KalachevDev is talking about. Fixed it.